doc/ref_arch/kubernetes/chapters/chapter02.md

```diff
@@ -23,7 +28,120 @@
 <a name="2.2"></a>
 ## 2.2 Reference Model Requirements
 
-Traceability to Reference Model.
+This reference model intends to implement both the Basic, Network Intensive and Compute Intensive Instance types defined in the Reference Model. The following table contains the requirements of the Reference Model and indicates if the Reference Architecture supports them.
+
+<a name="2.2.1"></a>
+### 2.2.1 Network Acceleration Extensions
+
+| Attribute | Description | Supported |
+|-----------|----------------------------------|-------|
+| .il-ipsec | In-line IPSec acceleration | optional <sup>1)</sup> |
+| .la-crypto | Look-Aside encryption/decryption engine | optional <sup>2)</sup> |
+| .la-trans | Look-Aside Transcoding acceleration | optional <sup>3)</sup> |
+| .la-programmable | Look-Aside programmable acceleration | optional <sup>4)</sup> |
+
+**1)** In Kubernetes based infrastructures virtio-ipsec can be applied only if the CNI plugin is OVS.<br>
+**2)** In Kubernetes based infrastructures virtio-crypto can be applied only if the CNI plugin is OVS.<br>
+**3)** In Kubernetes based infrastructures virtio-trans can be applied only if the CNI plugin is OVS.<br>
+**4)** In Kubernetes based infrastructures virtio-programmable can be applied only if the CNI plugin is OVS.
+
+
+<a name="2.2.2"></a>
+### 2.2.2 Network Interface Options
+
+| Attribute | Description | Supported |
+|-----------|----------------------------------|-------|
+| n1, n2, n3, n4, n5, n6 | | Y |
+| n10, n20, n30, n40, n50, n60 | | Y |
+| n25, n50, n75, n100, n125, n150 | | Y |
+| n50, n100, n150, n200, n250, n300 | | Y |
+| n100, n200, n300, n400, n500, n600 | | Y |
+
+<a name="2.2.3"></a>
+### 2.2.3 VIM Capabilities
+| Attribute | Description | Value | Supported |
+|-----------|---------------------------|-------|-------|
+| e.vim.cap.001 | Virtual Compute allocation | | Y |
+| e.vim.cap.002 | Virtual Storage allocation | | Y |
+| e.vim.cap.003 | Virtual Networking resources allocation | | Y |
+| e.vim.cap.004 | Multi-tenant isolation | | N <sup>1)</sup> |
+| e.vim.cap.005 | Images management | | Y |
+| e.vim.cap.006 | Virtual resources inventory per tenant | | Y |
+| e.vim.cap.007 | Resources Monitoring | | Y |
+| e.vim.cap.008 | Virtual resources Performance  |  | Y |
+| e.vim.cap.009 | Virtual resources Fault information | | Y |
+
+**1)** In a Kubernetes based infrastructure within one Kubernetes cluster multitenacy is provided only in resource management level. Isolation of execution environments requires separate Kubernetes clusters or worker nodes.
+
+<a name="2.2.4"></a>
+### 2.2.4 Instance Capabilities Mapping
+
+| Attribute | Description | Value | Supported |
+|-----------|---------------------------|-------|-------|
+| e.nfvi.res.cap.001 | Max number of vCPU that can be assigned to a single VNFC | 16 | Y |
```

Comment:
for e.nfvi.res.cap.001, e.nfvi.res.cap.002 and e.nfvi.res.cap.003 provide a comment as to why these limits exist?

Reply:
I explain this [here](https://github.com/cntt-n/CNTT/pull/738#discussion_r359199921).

Reply:
I've added a reference to the relevant RM section.